components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -218,12 +221,28 @@ const ContributorHighlightCard = ({
                   <BsLink45Deg size={22} />
                   <span>Copy link</span>
                 </DropdownMenuItem>
-                <DropdownMenuItem className="hidden rounded-md">
-                  <div className="flex gap-2.5 py-1  items-center pl-3 pr-7">
-                    <FaUserPlus size={22} />
-                    <span>Follow user</span>
-                  </div>
-                </DropdownMenuItem>
+                {loggedInUser ? (
+                  <DropdownMenuItem
+                    className={`rounded-md ${loggedInUser.user_metadata.user_name === user && "hidden"}`}
+                  >
+                    <div onClick={isError ? follow : unFollow} className="flex gap-2.5 py-1  items-center pl-3 pr-7">
+                      <FaUserPlus size={22} />
+                      <span>
+                        {!isError ? "Unfollow" : "Follow"} {user}
```

Comment:
So basically `isError` actually stands for whether the person is followed or not? If that's the case, it needs a clearer name from the hook itself to be able to quickly understand this.